docs/source/release-notes/4.0.0.rst

```diff
@@ -0,0 +1,50 @@
+4.0.0: 2023-04-18
+-----------------
+
+Backwards Incompatible Changes
+``````````````````````````````
+
+- :meth:`~github3.repos.comparison.Comparison.original_commits` Github now
```

Comment:
So we renamed the attribute from commits to original_commits and created a commits method that iterates over the API response from calling the method and paginates the results to get all commits. Want me to try to write that and update this?